### DISCUSSION TITLE
critique on the new update

### DISCUSSION THREAD

**Discussion by Patchwxrk** [keiwando/toonsquid#86] [Ideas]
I love the work you do, it's incredible you can do this all by yourself. A truly talented man you are, now I understand with you being a single person it is very hard to get to everyone's ideas, so feel free to store my idea for a future update instead

-i was very excited about tweening with the transform tool, but was sad to learn it did not work quite yet. if this could come out soon that would be incredible!

-I would like to be able to merge animation layers with each other like you can with pixel layers

-I would also like if there was an option to move the timeline to above, next to the buttons. i'm just used to that layout :)

that's all for now! great work

**Comment by Patchwxrk** [comment]
a few more i remembered

magic want
the ability to edit multiple frames with the lasso tool

**Comment by keiwando** [comment]
Thanks for all of the suggestions!
